src/components/button.scss

```diff
@@ -0,0 +1,217 @@
+/**
+ * @define default Button
+ */
+
+$Button-bg: $ui-white;
+$Button-bg-hover: $ui-background;
+$Button-color: $color-info;
+$Button-border-color: $ui-divider;
+$Button-border-color-hover: darken($Button-border-color, 10%);
+$Button-border-radius: $border-radius-medium;
+$Button-border-width: $border-width-small;
+$Button-border-bottom-width: $border-width-medium;
+$Button-disabled-opacity: 0.4;
+$Button-font-family: $base-font-family;
+$Button-font-size: $base-font-size;
+$Button-font-weight: $med-font-weight;
+$Button-height: 34px;
+$Button-line-height: $Button-height - 2;
+$Button-padding: 0 ($Button-height / 1.75);
+
+.Button {
+  background-color: $Button-bg;
+  border: $Button-border-color solid $Button-border-width;
+  border-bottom-width: $Button-border-bottom-width;
+  border-radius: $Button-border-radius;
+  color: $Button-color;
+  cursor: pointer;
+  display: inline-block;
+  font-family: $Button-font-family;
+  font-size: $Button-font-size;
+  font-weight: $Button-font-weight;
+  height: $Button-height;
+  line-height: normal;
+  margin: 0;
+  padding: $Button-padding;
+  text-decoration: none;
+  transition: all $duration-short ease-in-out;
+  vertical-align: middle;
+  white-space: nowrap;
+
+  &:hover,
+  &:active,
+  &:focus {
+    background-color: $Button-bg-hover;
+    border-color: $Button-border-color-hover;
+    outline: none;
+  }
+
+  &.is-active {
+    background-color: $Button-bg-hover;
+    box-shadow: 0 1px 0 $Button-border-color inset;
+  }
+
+  &.is-disabled,
+  &:disabled {
+    &,
+    &:hover,
+    &:active {
+      background-color: $Button-bg;
+      border-color: $Button-border-color;
+      cursor: not-allowed;
+      opacity: $Button-disabled-opacity;
+    }
+  }
+}
+
+/**
+ * Modifier: Button Primary
+ */
+
+$Button--primary-bg: $color-primary;
+$Button--primary-bg-hover: darken($Button--primary-bg, 5%);
+$Button--primary-color: $ui-white;
+$Button--primary-border-color: darken($Button--primary-bg, 7.5%);
+$Button--primary-border-color-hover: darken($Button--primary-bg, 10%);
+
+.Button--primary {
```

Comment:
Looking Good!  I like the positive / negative naming!

Can we cut out button--primary from here.  I know there are a few stragglers, but that was one we started to cut during the reskin.  It's in classic right now, because it's still in use in a few spots, but will be phased out.

![buttons](https://cloud.githubusercontent.com/assets/1189536/26112666/c772f56a-3a15-11e7-9588-8a83d4e79283.png)